runtime/src/lib.rs

```diff
@@ -497,6 +498,24 @@ impl nfts::Trait for Runtime {
     type Event = Event;
 }
 
+parameter_types! {
```

Comment:
are these parameters already discussed ?

Reply:
probably not yet discussed, however @philipstanislaus told me to just copy the same values as the parameter types of `utility` pallet

Reply:
These are the values I proposed a few weeks back in the deposit comparison. I have just triggered the discussion again on Slack, but they will most likely stay like this. Let's merge the PR and change them later if needed.